.devcontainer/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/rust/.devcontainer/base.Dockerfile
+
+# [Choice] Debian OS version (use bullseye on local arm64/Apple Silicon): buster, bullseye
+ARG VARIANT="buster"
+FROM mcr.microsoft.com/vscode/devcontainers/rust:0-${VARIANT}
+
+# [Optional] Uncomment this section to install additional packages.
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends \
+        python3 \
+        python3-venv \
+        python-is-python3 \
+        libclang-dev
+
+ENV VIRTUAL_ENV=/opt/venv
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN pip install --no-cache-dir \
+    "maturin>=0.9,<0.10" \
```

Comment:
Isn't the latest maturin version 0.13? We might even not tag a specific version at all, this container being a dev environment.

Reply:
I agree it doesn't need any constraints on the version at all. I originally didn't have the constraints but added them only after reading the pyoxigraph readme https://github.com/oxigraph/oxigraph/blob/main/python/README.md?plain=1#L49 where it stated:

> Maturin could be installed using the `pip install 'maturin>=0.9,<0.10'`.

I thought there was a special reason it had to be constrained to those version ranges. 

I'll remove the version constraints.

Reply:
Removed!